packages/block-library/src/comments/edit/placeholder.js

```diff
@@ -22,13 +21,8 @@ export default function PostCommentsPlaceholder( { postType, postId } ) {
 				.__experimentalDiscussionSettings
 	);
 
-	const disabledRef = useDisabled();
-
 	return (
-		<div
-			className="wp-block-comments__legacy-placeholder"
-			ref={ disabledRef }
-		>
+		<div className="wp-block-comments__legacy-placeholder" inert="true">
```

Comment:
Why exactly are we disabling this?

Reply:
This is just mirrors what existed before the PR. Basically we had a useDisabled hook that poly fills what "inert" does. So I'm not sure we changed anything in behavior here. I believe these "inert" within components are there because these blocks are "interactive", people can interact with them in the frontend but they are not meant to be interacted with in the backend (or navigated within). I'm not entirely sure that we really want to keep this visible for screen reader users, they are just mean to offer a visual representation of what the block will look like in the frontend but for interactions purposes, the intent is to hide these entirely. 

If we really want to leave a "trace", we may add a label maybe or something like that?

Reply:
@youknowriad That's the problem. We are giving sighted people information keyboard/screen reader users don't have access to. If the content on the back-end is really not important, let's add `display: none;` to it so it is inaccessible to everyone. Additional context make sense?

Thanks.

Reply:
I think I see what you're coming from but the reality is that the editor is also meant to be WYSIWYG and that concept doesn't really translate into screen readers. Sighted users will see something different from the backend and frontend and consider this a visual bug. 

In terms of interactions, I agree, both sighted and screen reader users should have the exact same thing but the visuals are important for sighted users.

Would it help if we add a label or something saying "Visual Preview of block/whatever we're previewing" or something like that?

Reply:
@youknowriad 

> Also, this is independent of the use of inert, it has always been this way in the editor.

That's got to change but I guess it can go into an issue for further discussion.

Labels don't help here because the preview itself is still not communicated equally.

Why is it not possible to disable interactions with blocks in an accessible way? #54322 accomplishes just that. People with vision can't interact with the block and screen readers can still see the block content but cannot interact with the block.

We've got to quit caring so much about this front-end vs. back-end. Honestly, do you think anyone at all would notice the usage of `inert` vs. `aria-disabled` with event listeners to disable a control? Absolutely not. It just makes it more work for developers and that I fear is the real problem here.

Reply:
Below example shows how this can still remain accessible but preventing the form from actually working.

```html
<html>
	<head>
		<title>Test</title>
	</head>
	<body>
		<h1>With inert</h1>
		<form class="form1" action="https://www.google.com" method="GET" inert>
			<input type="search" value="" name="search1" aria-label="Search 1" placeholder="Search 1" />
			<input type="submit" value="Search 1" />
		</form>
		<br />
		<h1>Accessible way but more work</h1>
		<form class="form2" action="https://www.google.com" method="GET">
			<input type="search" value="" name="search2" aria-label="Search 2" placeholder="Search 2" aria-disabled="true" />
			<input type="submit" value="Search 2" aria-disabled="true" />
		</form>
		<script type="text/javascript">
			const form2 = document.querySelector('.form2');
			form2.addEventListener('submit', function(e) {
				e.preventDefault();
			} );
		</script>
	</body>
</html>
```

Reply:
I've created an issue where can record discussions and debate alternatives:

- https://github.com/WordPress/gutenberg/issues/54369

Reply:
Now that I think about it, all of our usage of "inert" can be seen as "screenshots". In that sense, I think maybe the solution going forward is to use a role="img" on these elements with a label. When you show a preview of a block or a template, it's just like an image.